### PR TITLE
增加nodelocaldns，触发重新构建k8s-build-image

### PR DESCRIPTION
**Pull request by zhangguanzhang**

### CODE REVIEW

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/wall |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/chage |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chsh |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/passwd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/gpasswd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chsh |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/newgrp |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/mount |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/wall |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/gpasswd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/wall |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/mount |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/wall |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/newgrp |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/wall |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chsh |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/passwd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/sbin/unix_chkpwd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/newgrp |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/wall |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/gpasswd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chfn |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chsh |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/newgrp |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/expiry |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-iptables:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/gpasswd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/debian-base:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0007 | FATAL | Always combine 'apt-get update' with 'apt-get install|upgrade' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0005 | FATAL | Use 'rm -rf /var/lib/apt/lists' after 'apt-get install|update' : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0003 | WARN | Avoid dist-upgrade in container : |0 /bin/sh -c apt-get update     && apt-get dist-upgrade -y |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/chsh |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setgid file: grwxr-xr-x usr/bin/expiry |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-dns/k8s-dns-node-cache:1.22.28

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/gpasswd |

**Comment by qiangxuhui** [comment]
- cr.loongnix.cn/k8s-build-image/go-runner:latest

| code | level | alerts |
| --- | --- | --- |
| CIS-DI-0001 | WARN | Last user should not be root |
| DKL-DI-0006 | WARN | Avoid 'latest' tag |
| CIS-DI-0005 | INFO | export DOCKER_CONTENT_TRUST=1 before docker pull/build |
| CIS-DI-0006 | INFO | not found HEALTHCHECK statement |
| CIS-DI-0008 | INFO | setuid file: urwxr-xr-x usr/bin/mount |

File: kubernetes/nodelocaldns/1.22.28/Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+
+build: src/$(SOURCE)
+	docker run --rm -e http_proxy=$(http_proxy) -e https_proxy=$(https_proxy) \
+	  -v `pwd`/src/$(SOURCE):/v -w /v \
+	  -e GOARCH=loong64 -e CGO_ENABLED=0 \
+	  $(GO_IMG) \
+	  go build -o bin/loong64/node-cache -ldflags "-X k8s.io/dns/pkg/version.VERSION=$(TAG)" \
+	  cmd/node-cache/main.go
+image: build
```
Comment (zhangguanzhang) [inline kubernetes/nodelocaldns/1.22.28/Makefile:34]:
```
make containers CONTAINER_BINARIES=node-cache DOCKER_BUILD_FLAGS=--load VERSION=1.22.28
```

**Pull request merged**